ct-app/postman/postman_tasks.py

```diff
@@ -58,13 +67,13 @@ async def async_send_1_hop_message(
     if address is None:
         log.error("Could not get connect to node. Trying with a backup node")
 
+        node_address, node_index = get_next_node_address(node_list, node_index)
+
         # if there are no more nodes in the list, the task is stopped
-        if node_index == len(node_list) - 1:
+        if not node_address:
             log.info("This is the last node in the list, stopping")
```

Comment:
If this is a failure, the log should be `error`, not `info`.

Reply:
Agreed. Resolved in new commit.